scalingo.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "Sample Scala Scalatra",
-    "description": "Sample Scala web application using the Scalatra framework",
-    "logo": "https://scalingo.com/logo.svg",
-    "repository": "https://github.com/Scalingo/sample-scala-scalatra",
-    "website": "http://sample-scala-scalatra.scalingo.io"
+  "name": "Sample Scala Scalatra",
+  "description": "Sample Application with Scala and Scalatra",
```

Comment:
Well, I find the old description more accurate

Reply:
Maybe mix them